packages/connect/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@substrate/connect",
-  "version": "0.6.5",
+  "version": "0.7.0",
```

Comment:
Could we bump the version in another PR maybe? For proper granularity, so that we know exactly which commit is published on NPM

Reply:
Yeah! For sure!